coursedata/texts/en.yaml

```diff
@@ -47,6 +47,7 @@ ClientErrorMessages:
       - "Well done!"
       - "Excellent!"
       - "You did great!"
+    Transpile_incomplete: "Your code works, but you forgot to use the newly learned subjects!"
```

Comment:
Can we replace subjects by 'commands' here? That is the word we commonly use. Also... can you add an nl translation too? In principle, the Hedy code team delivers all features in nl and en.

Reply:
Solved with commit: '#1219 Dutch translation transpile_incomplete'